e2e-tests/cypress/integration/project-application-documents.feature

```diff
@@ -154,4 +154,35 @@ Feature: Project Application documents page
             | 7                     |
             | 8                     |
         Then I verify text "Showing 141 to 152 of 152 results" is present on the page
-        Then I verify that only "12" results present on each page
\ No newline at end of file
+        Then I verify that only "12" results present on each page
+
+    Scenario: verify documents available for project is in descending order by date published
+        Given I navigate to "Hinkley Point C New Nuclear Power Station Material Change 1" project Overview page
+        When I click on "Project application documents" link
+        Then I can verify that the project documents displayed in descending order
+            | Data       |
```

Comment:
`Date` would be a better column name and I think it would be good to also have another column - `Title` which we can use to verify the correct document is shown